modules/channel-twilio/src/backend/index.ts

```diff
@@ -0,0 +1,46 @@
+import 'bluebird-global'
+import * as sdk from 'botpress/sdk'
+
+import en from '../translations/en.json'
+import fr from '../translations/fr.json'
+
+import { TwilioService } from './twilio'
+
+let service: TwilioService
+
+const onServerStarted = async (bp: typeof sdk) => {
+  service = new TwilioService(bp)
+  try {
+    await service.initialize()
+  } catch (err) {
+    bp.logger.attachError(err).warn('Channel misconfigured')
+  }
+}
+
+const onBotMount = async (bp: typeof sdk, botId: string) => {
+  await service.mountBot(botId)
+}
+
+const onBotUnmount = async (bp: typeof sdk, botId: string) => {
+  await service.unmountBot(botId)
+}
+
+const onModuleUnmount = async (bp: typeof sdk) => {
+  bp.events.removeMiddleware('twilio.sendMessages')
+  bp.http.deleteRouterForBot('channel-twilio')
+}
+
+const entryPoint: sdk.ModuleEntryPoint = {
+  onServerStarted,
+  onBotMount,
+  onBotUnmount,
+  onModuleUnmount,
+  translations: { en, fr },
```

Comment:
This object is only to provide translations for the frontend

Reply:
Understand that, but since we also need to access it in `twilio.ts` when rendering the localized prefix for the outgoing messages that contain options, ie: dropdowns, single-choice, etc. We are reusing the same mechanism until you guys provide a better API for it as discussed [here](https://github.com/botpress/botpress/pull/3487#discussion_r443864527).